src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveClaimantResponseTest.java

```diff
@@ -108,6 +108,22 @@ public void shouldSaveClaimantResponseRejection() throws Exception {
         assertThat(claimantResponse.getAmountPaid().orElse(null)).isEqualTo(BigDecimal.TEN);
     }
 
+    @Test
+    public void shouldSaveClaimantResponseAcceptationWithoutFormaliseOptionOnStatePaid() throws Exception {
+        ClaimantResponse response = builder().buildStatePaidAcceptationWithoutFormaliseOption();
+
+        makeRequest(claim.getExternalId(), SUBMITTER_ID, response)
+            .andExpect(status().isCreated());
+
+        Claim claimWithClaimantResponse = claimStore.getClaimByExternalId(claim.getExternalId());
+
+        assertThat(claimWithClaimantResponse.getClaimantRespondedAt().isPresent()).isTrue();
+
+        ResponseAcceptation claimantResponse = (ResponseAcceptation) claimWithClaimantResponse.getClaimantResponse()
```

Comment:
As `claimantResponse` is not being used here. I assume the purpose of this is to verify whether claimantResponse exists on the object. Better option would be:
```
assertThat(claimWithClaimantResponse.getClaimantResponse()).isPresent();
``` 
You may want to check the type is acceptation:

```
ClaimantResponse claimantResponse = claimWithClaimantResponse.getClaimantResponse().orElseThrow(AssertionError::new);
        assertThat(claimantResponse.getType()).isEqualTo(ACCEPTATION);
```